api/tests/tooling/domain-builder/factory/build-certification-candidate.js

```diff
@@ -69,6 +76,7 @@ buildCertificationCandidate.pro = function ({
   sessionId = 456,
   complementaryCertification = null,
   billingMode = 'FREE',
+  subscriptions = [buildCoreSubscription()],
```

Comment:
Préférable d'importer le domainBuilder plutôt que la méthode directement ?

```suggestion
subscriptions = [domainBuilder.buildCoreSubscription()],
```

Reply:
C'est remplacé dans un autre commit :)